src/comp/comp.py

```diff
@@ -20,52 +20,100 @@ def __repr__(self):
     Human("Igon", 41),
     Human("David", 31),
 ]
+# print(humans)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
-print("Starts with D:")
-a = []
+print("Starts with D: ")
+a = [d.name for d in humans if d.name[0] == 'D']
```

Comment:
Awesome job on the list comprehensions! You should have a pretty thorough understanding of these now! ;)
<img width="1657" alt="Screen Shot 2019-12-09 at 2 42 19 AM" src="https://user-images.githubusercontent.com/16110931/70416674-9abab800-1a2d-11ea-9db2-e68b2ee1b109.png">